File: wtplayer/src/WTLight.cpp
#include "basetypes.hpp"
#include "WTLight.hpp"

WTLight::WTLight(WT* wt_,
                 int Type) :
  WTContainer(wt_) {
}

void WTLight::setColor(int Red,
                       int Green,
                       int Blue){
  APILOG;
}

float WTLight::getConstantAttenuation(){
  APILOG;
  return 0;
}

float WTLight::getLinearAttenuation(){
  APILOG;
  return 0;
}

float WTLight::getPenumbra(){
  APILOG;
  return 0;
}

float WTLight::getQuadraticAttenuation(){
  APILOG;
  return 0;
}

float WTLight::getRange(){
  APILOG;
  return 0;
}

float WTLight::getUmbra(){
  APILOG;
  return 0;
}

void WTLight::setConstantAttenuation(float Constant_Attenuation){
  APILOG;
}

void WTLight::setLinearAttenuation(float Linear_Attenuation){
  APILOG;
}

void WTLight::setPenumbra(float Degrees){
  APILOG;
}

void WTLight::setQuadraticAttenuation(float Quadratic_Attenuation){
  APILOG;
}

void WTLight::setRange(float Range){
  APILOG;
}

void WTLight::setUmbra(float Degrees){
  APILOG;
}

bool WTLight::isStatic(){
  APILOG;
  return 0;
}

void WTLight::setLightMask(int LightMask){
  APILOG;
}

int WTLight::getLightMask(){
  APILOG;
  return 0;
}

File: wtplayer/javasrc/wildtangent/webdriver/WTLight.java
package wildtangent.webdriver;

import wildtangent.webdriver.WTContainer;

public interface WTLight
extends WTContainer {
    public float getConstantAttenuation();

    public float getQuadraticAttenuation();

    public float getRange();

    public void setConstantAttenuation(float var1);

    public void setQuadraticAttenuation(float var1);

    public void setRange(float var1);

    public boolean isStatic();

    public float getPenumbra();

    public float getUmbra();

    public void setColor(int var1, int var2, int var3);

    public void setPenumbra(float var1);

    public void setUmbra(float var1);

    public float getLinearAttenuation();

    public void setLinearAttenuation(float var1);

    public void setLightMask(int var1);

    public int getLightMask();
}

